### PR TITLE
Remove the Mark all Read

### PR DESCRIPTION
**Pull request by plytro**
It's a popular request

https://thismight.be/offensive/?c=comments&fileid=391793#284437278

### CODE REVIEW

File: styles/oldskool.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 body {
-	background: orange;
+	background: #336;
```
Comment (numist) [inline styles/oldskool.css]: This needs to get fixed up before merge

**Comment by numist** [comment]
@themaxx: opinions?

**Comment by themaxx** [comment]
Fine with me. I never use it, occasionally accidentally click it, and I think I originally implemented it just so I could reset the list during development/testing. (At very least it should be moved away from the list itself and/or have a confirmation.) But I’m fine with its removal.

On May 28, 2015 at 3:20:15 PM, Scott Perry (notifications@github.com) wrote:

@themaxx: opinions?

**Comment by numist** [comment]
Yeah I always thought it violated [Fitt's Law](http://en.wikipedia.org/wiki/Fitts's_law).

Thanks @plytro!

**Pull request merged**